packages/react/src/components/Tag/Tag.js

```diff
@@ -35,11 +43,11 @@ const Tag = ({ children, className, type, filter, disabled, ...other }) => {
   return filter ? (
     <span
       className={tagClasses}
-      title="Clear filter"
+      title={title ? title : 'Clear filter'}
```

Comment:
alternatively we can set a default value for `title` or do

```suggestion
      title={title || 'Clear filter'}
```

to avoid ternary operator

same on line 50

Reply:
Just pushed!

Reply:
wait @joshblack line 50 as well